<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" columnstretch="0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Init from focus:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="read1stFocusComboBox"/>
     </item>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="read2ndFocusComboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="readFocusButton">
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="TableView" name="opticsTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
</ui>
